client/src/utils/dateUtils.ts

```
@@ -90,34 +89,38 @@ export const getDateRangeFromFilter = (filter: string): DateRange => {
 
     case "Last month":
       // Get start and end of previous month
       startDate = new Date(now.getFullYear(), now.getMonth() - 1, 1);
       endDate = new Date(now.getFullYear(), now.getMonth(), 0); // Last day of previous month
       startDate.setHours(0, 0, 0, 0);
-      endDate.setHours(23, 59, 59, 999);
       break;
 
     case "This Quarter":
       // Get start of current quarter
       const currentQuarter = Math.floor(now.getMonth() / 3);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)